docs/plugins/shiki.md

```diff
@@ -86,3 +94,246 @@ export default {
 
 - Also see:
   - [Shiki > Transformers](https://shiki.style/guide/transformers)
+
+### lineNumbers
+
+- Type: `boolean | number`
+
+- Default: `true`
+
+- Details:
+
+  Configure code line numbers.
+
+  - If the option is set to `true` , the line numbers will be enabled.
+  - If the option is set to `false` , the line numbers will be disabled.
+  - If the option is set to `number` ，is the minimum number of lines to enable line numbers。
+    For example, if you set it to 4, line numbers will only be enabled when your code block has at least 4 lines of code.
+
+  You can add `:line-numbers` / `:no-line-numbers` mark in your fenced code blocks to override the value set in config.
+
+**Input**
```

Comment:
```suggestion
**Input**:
```

I think column is inserted in most places of the documentation you can confirm that